reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -43,4 +65,6 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
             return LocalTime(hour, minute)
         }
     }
+
+    private val MINUTES_PER_HOUR = 60
```

Comment:
Move this into companion object and make it const
private const val MINUTES_PER_HOUR = 60